sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1095,7 +1099,7 @@ static int sof_card_dai_links_create(struct device *dev,
 		    group_generated[endpoint->group_id])
 			continue;
 
-		ret = create_sdw_dailink(dev, &be_id, links, sdw_be_num,
+		ret = create_sdw_dailink(card, dev, &be_id, links, sdw_be_num,
```

Comment:
this increases the number of `create_sdw_dailink()` arguments from 13 to 14...

Reply:
It's rather horrible indeed @lyakh. One of these days we are going to have to rewrite this code anyways, but that's a much bigger PR.